### PR TITLE
Add footer using Nuxt UI and Tailwind

### PR DESCRIPTION
**Pull request by patrickpeinanw**
The Tooltip is working but the pop-up text box still have the default style of Nuxt UI.

### CODE REVIEW

File: btr-web/btr-layouts/components/footer/BcrosFooter.vue [hunk 1]
```diff
@@ -1,12 +1,73 @@
...
+            <UIcon class="text-2xl tooltip-icon" name="i-mdi-information-outline" />
+          </div>
+        </UTooltip>
+      </nav>
+    </div>
+  </footer>
 </template>
 
 <script lang="ts">
+import pkg from '../../package.json'
```
Comment (hfekete) [inline btr-web/btr-layouts/components/footer/BcrosFooter.vue:33]: Could you move this to parent, and get it here as a prop, like we chatted ?

File: btr-web/btr-layouts/components/footer/BcrosFooter.vue [hunk 2]
```diff
@@ -1,12 +1,73 @@
...
 </script>
 
 <style scoped>
+.app-footer {
+  border-top: 2px solid #fcba19;
+  background-color: #003366;
+  font-size: 0.9735rem;
+}
+
+.tooltip-icon {
```
Comment (hfekete) [inline btr-web/btr-layouts/components/footer/BcrosFooter.vue:52]: Colors are ok for now, but should be put in a common place. I have created a ticket for this

[extract colors ticket](https://app.zenhub.com/workspaces/registries-benowner-644043015b6fa1002c69a9f4/issues/gh/bcgov/entity/18126)

File: btr-web/btr-layouts/components/footer/BcrosFooter.vue [hunk 3]
```diff
@@ -1,12 +1,73 @@
...
+  color: #8099b3;
+}
+
+@media (min-width: 960px) {
```
Comment (hfekete) [inline btr-web/btr-layouts/components/footer/BcrosFooter.vue:56]: Did it end up having effect ? I remember some margins changed, when you were showcasing it, but I do not remember if it had any visible effect for human thats reading it.

**Review by hfekete — CHANGES_REQUESTED** [review]

**Review by kialj876 — CHANGES_REQUESTED** [review]
Hey patrick I've got few comments for ya. Also can you please add a basic unit test for this component

File: btr-web/btr-layouts/components/footer/BcrosFooter.vue (hunk 3 above)
Comment (hfekete) [reply btr-web/btr-layouts/components/footer/BcrosFooter.vue:51]: Lets remove it for now, and see if we need to add it down the line. We should probably add the separators for all sizes.
As if I remember we do not work towards supporting different screen sizes.

You can do something like border-l-2 which is border left 2px, except on the last element :)

File: btr-web/btr-layouts/components/footer/BcrosFooter.vue [hunk 4]
```diff
@@ -29,17 +19,22 @@
...
-import pkg from '../../package.json'
-
-export default {
-  setup () {
-    const appVersion = pkg.version
-    return {
-      appVersion
-    }
+<script setup lang="ts">
+const { appVersion } = defineProps({
```
Comment (kialj876) [inline btr-web/btr-layouts/components/footer/BcrosFooter.vue:23]: For this case you can simply do this:
`defineProps<{appVersion: string}>()`
It's a required value so you don't need a default and you can access props directly in the dom

File: btr-api/migrations/versions/2049364ea0a6_.py [hunk 5]
```diff
@@ -0,0 +1,71 @@
+"""empty message
```
Comment (kialj876) [inline btr-api/migrations/versions/2049364ea0a6_.py:1]: Hey patrick can you fix your branch so that it doesn't include hrvoje's stuff as a new commit ?

**Review by kialj876 — CHANGES_REQUESTED** [review]

File: btr-api/migrations/versions/2049364ea0a6_.py (hunk 5 above)
Comment (kialj876) [reply btr-api/migrations/versions/2049364ea0a6_.py:1]: if you need help rebasing let me know

**Review by hfekete — APPROVED** [review]

**Pull request merged**